service/src/main/java/com/codeforcommunity/ServiceMain.java

```diff
@@ -103,6 +87,7 @@ private void initializeServer() {
     ITeamsProcessor teamsProc = new TeamsProcessorImpl(this.db);
     IProtectedSiteProcessor protectedSiteProc = new ProtectedSiteProcessorImpl(this.db);
     ISiteProcessor siteProc = new SiteProcessorImpl(this.db);
+    IInactiveUserProcessor inactiveProc = new InactiveUserProcessorImpl(this.db, emailer);
```

Comment:
So far, we have been putting emails in the processor to which they are most related. Password and user emails are in the protected user processor so it may make sense to add this function to the ProtectedSite processor

Reply:
Moved to protectedSite processor 👍